### PR TITLE
Automatic release and version update

### LINKED ISSUE #49: Find a way to freaking automate the release
Optimist : 01h00
Pessimist : 05h00
Estimated time: 03h00
Real time: 04h00

### PR DESCRIPTION
**Pull request by clemdiep-heig**
I have tested it in another project, all seems to be fine.

It will update the version in `pom.xml` and `version.txt` if needed.\
The release message should contains all pull request title has changelog.

closes #49

### CODE REVIEW

**Review by dylanrodr-heig — CHANGES_REQUESTED** [review]
I tried to publish a release after forking the repository. But it didn't seemed to work.
Maybe I'm using it wrong :0

[fork link](https://github.com/dylanrodr-heig/dil-test/releases/tag/v1.0.2)

**Review by dylanrodr-heig — APPROVED** [review]
Good job ! 🥇

File: .github-cli/update-version [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+git config --global user.email $EMAIL
+git config --global user.name $NAME
```
Comment (Isotech42) [inline .github-cli/update-version:4]: Hum.. I don't understand where and why you have to use these $EMAIL and $NAME ?
I think we can delete these lines
```suggestion

```

**Review by Isotech42 — CHANGES_REQUESTED** [review]

File: .github-cli/update-version (hunk 1 above)
Comment (clemdiep-heig) [reply .github-cli/update-version:4]: Ho yes, it was some old lines from previous attempts !

**Review by Isotech42 — APPROVED** [review]
For me it's all perfect ! Thank you ! 🍻  (except that there is spaces...)

**Pull request merged**